pkg/api/alerting.go

```diff
@@ -29,7 +29,7 @@ func GetAlertStatesForDashboard(c *m.ReqContext) Response {
 	dashboardID := c.QueryInt64("dashboardId")
 
 	if dashboardID == 0 {
-		return ApiError(400, "Missing query parameter dashboardId", nil)
+		return Error(400, "Missing query parameter dashboardId", nil)
```

Comment:
Why should `Api` be removed?

Reply:
`golint pkg/api/... | grep stutt` gives:
```
pkg/api/common.go:79:6: func name will be used as api.ApiSuccess by other packages, and that stutters; consider calling this Success
pkg/api/common.go:85:6: func name will be used as api.ApiError by other packages, and that stutters; consider calling this Error
```

Reply:
Makes sense :)

Reply:
It's also more consistent because other similar methods don't have the `Api` prefix.

Reply:
hm.. then rename it to JsonError or NewErrorRsp, the api prefix is to make it clear the response is json and is only to be used in `/api/` http handlers.  the api package is not only for the http api but also some other http handlers (maybe the package should be named http but that conflicts with standard goland package name)

Reply:
Well, to be consistent, we had to rename several methods to `New***Response` that returns a new `*NormalResponse` instance. Thanks for merging.